### PR TITLE
ws in controller sample

### PR DESCRIPTION
**Pull request by fiyazbinhasan**
Fixes https://github.com/dotnet/AspNetCore.Docs/issues/22747

@Rick-Anderson added a sample for .net 6 

[Internal review URL](https://review.docs.microsoft.com/en-us/aspnet/core/fundamentals/websockets?view=aspnetcore-5.0&branch=pr-en-us-22760#accept-websocket-requests)

### CODE REVIEW

**Comment by Rick-Anderson** [comment]
@trampster please review.

File: aspnetcore/fundamentals/websockets.md [hunk 1]
```diff
@@ -77,6 +77,10 @@ The following example is from later in the `Configure` method:
 
 A WebSocket request could come in on any URL, but this sample code only accepts requests for `/ws`.
 
+To keep the `Startup` clutter-free, a similar approach can be taken in an action method:
+
+[!code-csharp[](websockets/samples/6.x/WebSocketsSample/Controllers/WebSocketController.cs?name=WebSocketController)]
```
Comment (Rick-Anderson) [inline aspnetcore/fundamentals/websockets.md:82]:
```suggestion
[!code-csharp[](websockets/samples/6.x/WebSocketsSample/Controllers/WebSocketController.cs?name=snippet)]
```

**Review by Rick-Anderson — CHANGES_REQUESTED** [review]

**Review by Rick-Anderson — APPROVED** [review]

File: aspnetcore/fundamentals/websockets.md [hunk 2]
```diff
@@ -77,6 +77,10 @@ The following example is from later in the `Configure` method:
 
 A WebSocket request could come in on any URL, but this sample code only accepts requests for `/ws`.
 
+To keep the `Startup` clutter-free, a similar approach can be taken in an action method:
```
Comment (BrennanConroy) [inline aspnetcore/fundamentals/websockets.md:80]:
```suggestion
To keep the `Startup` clutter-free, a similar approach can be taken in a controller method:
```

>To keep the `Startup` clutter-free

I don't know if this is needed. It implies that controllers are the only way to remove code from startup which is wrong.

Comment (Rick-Anderson) [reply aspnetcore/fundamentals/websockets.md:80]:
```suggestion
A similar approach can be taken in an controller method:
```

File: aspnetcore/fundamentals/websockets.md [hunk 3]
```diff
@@ -77,6 +77,10 @@ The following example is from later in the `Configure` method:
 
 A WebSocket request could come in on any URL, but this sample code only accepts requests for `/ws`.
 
+A similar approach can be taken in an controller method:
```
Comment (BrennanConroy) [inline aspnetcore/fundamentals/websockets.md:80]:
```suggestion
A similar approach can be taken in a controller method:
```

**Review by BrennanConroy — APPROVED** [review]

File: aspnetcore/fundamentals/websockets.md (hunk 3 above)
Comment (fiyazbinhasan) [inline aspnetcore/fundamentals/websockets.md:80]: a controller method

**Pull request merged**